.circleci/config.yml

```diff
@@ -40,74 +40,22 @@ jobs:
           command: npx bundlesize
 
   danger:
-    docker:
-      - image: circleci/node:10.13
+    executor: node/build
     steps:
       - yarn/setup
       - run:
           name: Danger
           command: DANGER_GITHUB_API_TOKEN="15e52de81a772b174cc5""e1813d0083564c69c325" yarn danger ci
 
   test:
-    docker:
-      - image: circleci/node:10.13
+    executor: node/build
     parallelism: 2
     steps:
       - yarn/setup
       - run:
           name: Test
           command: COMMIT_HASH_FULL=$(git rev-parse HEAD) CODECOV_TOKEN=$CODECOV_TOKEN BRANCH_NAME=$CIRCLE_BRANCH yarn test
 
-  push_staging_image: &push_image
-    docker:
-      - image: artsy/hokusai:0.5
-    steps:
-      - add_ssh_keys
-      - checkout
-      - setup_remote_docker
-      - run:
-          name: Push
-          command: |
-            hokusai registry push --tag $CIRCLE_SHA1 --force --overwrite
-  push_production_image: *push_image
-
-  deploy_hokusai_staging:
-    docker:
-      - image: artsy/hokusai:0.5
-    steps:
-      - add_ssh_keys
-      - checkout
-      - setup_remote_docker
-      - attach_workspace:
-          at: /tmp/workspace
-      - run:
-          name: Configure
-          command: hokusai configure --kubectl-version 1.10.7 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
-      - run:
-          name: Update staging branch
-          command: git push git@github.com:artsy/force.git $CIRCLE_SHA1:staging --force
-      - run:
-          name: Deploy
-          command: hokusai staging deploy $CIRCLE_SHA1
-          no_output_timeout: 20m
-
-  deploy_hokusai_production:
-    docker:
-      - image: artsy/hokusai:0.5
-    steps:
-      - add_ssh_keys
-      - checkout
-      - setup_remote_docker
-      - attach_workspace:
-          at: /tmp/workspace
-      - run:
-          name: Configure
-          command: hokusai configure --kubectl-version 1.10.7 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
-      - run:
-          name: Deploy
-          command: hokusai production deploy $CIRCLE_SHA1 --git-remote origin
-          no_output_timeout: 20m
```

Comment:
I think we should add this to the orb as well @zephraph

Reply:
The `no_output-timeout` thing?